modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,21 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="gitops-dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+Argo CD cannot map to users if they have a direct `ClusterRoleBinding` role. You can manually change users' role as `role:admin` on SSO through OpenShift.
```

Comment:
This sentence implies that Argo CD is mapping to users, but the actual idea is that Argo CD maps users to specific roles. Suggest something to the effect: cannot map to users/cannot map users to specific roles if...
s/a user's/users' role